doc/conf.py

```diff
@@ -123,6 +130,93 @@
 # pixels large.
 #html_favicon = None
 
+# Set up favicons with sphinx-favicon. 
```

Comment:
```suggestion
# Set up favicons with sphinx-favicon.
```

There are some trailing whitespaces in the patch:
```
$ flake8 tarantool | grep "blank line contains whitespace"
tarantool/connection.py:1549:1: W293 blank line contains whitespace
tarantool/mesh_connection.py:106:1: W293 blank line contains whitespace
tarantool/msgpack_ext/types/interval.py:83:1: W293 blank line contains whitespace
tarantool/msgpack_ext/types/datetime.py:148:1: W293 blank line contains whitespace
$ flake8 tarantool/ | grep trailing
tarantool/connection_pool.py:893:59: W291 trailing whitespace
tarantool/msgpack_ext/decimal.py:144:23: W291 trailing whitespace
tarantool/msgpack_ext/decimal.py:266:69: W291 trailing whitespace
tarantool/msgpack_ext/decimal.py:351:20: W291 trailing whitespace
tarantool/msgpack_ext/types/interval.py:140:69: W291 trailing whitespace
```

Reply:
I have fixed all doc-related trailing whitespaces. I decided not to change code-related whitespaces for now so this PR wouldn't became even more complicated than now.

It's a shame we still doesn't have a linter for such things, but I think you've seen how many errors and warnings should be fixed before adding this CI step.